superfaktura/utils/__init__.py

```diff
@@ -0,0 +1,10 @@
+import tempfile
+from typing import IO
+
+
+def save_temporary_file_as_pdf(
```

Comment:
This is very specific function just or an example and thus should not be in utils (it is strange, that it closes the input stream, for example)